app/src/org/commcare/android/database/user/UserDatabaseUpgrader.java

```diff
@@ -208,15 +221,71 @@ private boolean upgradeSevenEight(SQLiteDatabase db, int oldVersion, int newVers
         }
     }
 
-    /**
+    /*
      * Deserialize user fixtures in db using old form instance serialization
      * scheme, and re-serialize them using the new scheme that preserves
      * attributes.
      */
-    private boolean upgradeEightNine(SQLiteDatabase db, int oldVersion, int newVersion) {
+    private boolean upgradeEightNine(SQLiteDatabase db) {
         return FixtureSerializationMigration.migrateFixtureDbBytes(db, c);
     }
 
+    /**
+     * Adding an appId field to FormRecords, for compatibility with multiple apps functionality
+     */
+    private boolean upgradeNineTen(SQLiteDatabase db) {
+        db.beginTransaction();
```

Comment:
This could potentially take a long time. I'd probably request more time from the commcare service connector so the app doesn't time out trying to connect to the db (I think the fixture migration code does this)